tutorials/polkadot-sdk/parachains/local-chain/connect-default-nodes.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Simulate a Network
-description: Create a private blockchain network with authorized validators using Polkadot SDK. Start nodes, connect peers and verify block production.
+title: Connect Default Nodes
```

Comment:
This is cool, then we can create a new section to connect to custom nodes if needed

Reply:
By default I meant "Alice, Bob" and the section just below it is how to add your own nodes. unsure of a better name

![Uploading image.png…]()

Reply:
well, i like default, unsure if it is 100% accurate

Throughout the tutorial we use "two predefined accounts". Maybe "Connect Nodes With Predefined Accounts"? it's super long tho

Reply:
Yeah I tried that but it is way too long maybe for the left hand side nav we can use the short version but for the title of the page we can use the long one

Reply:
Yeah, that sounds good to me as long as it’s not counterproductive for SEO